src/scss/_drop-down.scss

```diff
@@ -0,0 +1,97 @@
+// sass-lint:disable no-qualifying-elements
+/// @access private
+/// @outputs styling for primary navigation drop down menus
+@mixin _oHeaderServicesDropDown() {
+	.o-header-services {
+		li[data-o-header-services-level] {
+			position: relative;
+		}
+
+		li[data-o-header-services-level] > a {
+			padding-right: oTypographySpacingSize(10);
+
+			&:after {
+				@include oIconsGetIcon('arrow-down', _oHeaderServicesGet('nav-text'), 24px);
+				content: '';
+
+				padding-left: oTypographySpacingSize(4);
+				position: absolute;
+				top: $_o-header-services-padding;
+				transition: transform 0.2s ease;
+			}
+		}
+
+		ul[data-o-header-services-level] {
+			@include oTypographySize(-1);
+			background: _oHeaderServicesGet('nav-background');
+			border: 1px solid _oHeaderServicesGet('nav-border');
+			box-sizing: border-box;
+			min-width: auto;
+			padding-left: 0;
+			position: absolute;
+			opacity: 0;
+			transform: translateY(-100%);
+			transition:
+				opacity 0.01s linear,
+				transform 0.2s ease-in;
+			z-index: -1;
+
+			&.o-header-services__list--right {
+				right: 0;
+			}
+		}
+
+		.core li[data-o-header-services-level]:hover,
```

Comment:
😂 ![kapture 2019-01-23 at 17 26 06](https://user-images.githubusercontent.com/10405691/51624996-0a1ddb00-1f34-11e9-9fdb-0653767abcc8.gif)

Maybe the arrows can be hidden for core?

Reply:
My mistake, the menu is supposed to drop down on hover for core. its fixed now.